spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMetadataTables.java

```diff
@@ -488,6 +493,60 @@ public void testMetadataLogEntries() throws Exception {
         metadataLogWithProjection);
   }
 
+  @Test
```

Comment:
Sorry can we also change the test name to capture the scenario better, maybe testFilesTableTimeTravelWithSchemaEvolution  .  I know its a bit long, couldnt think anything shorter

Reply:
Not saying no to good suggestions :)

Reply:
Haha, hmmm - I'd argue that the first one is more descriptive of the problem and better describes the motivation in having the specific test, even though it's longer... testFilesTimeTravelAddedColumn is a lot more literal

Reply:
No problem, up to you!